woocommerce/woocommerce
# CYS - Core: fix Product Rating block renders

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This regression has been introduced with https://github.com/woocommerce/woocommerce/pull/45196, and it is reproducible only when JetPack is installed. 
I'm not sure why this happens, but I guess that `getEditedPostType` triggers the rendering of the Editor, and this fixes the issue.



Closes #45593.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure you have the WooCommerce Beta Tester plugin installed and activated.
2. Ensure you have Jetpack plugin installed.
3. Under Tools > WCA Test Helper > Features, make sure customize-store is enabled.
4. Under Tools > WCA Test Helper > Tools, click on Reset Customize Your Store.
5. Head over to WooCommerce > Home > Customize your store.
6. Click on "Start designing" and proceed to the Pattern Assembler.
7. Ensure that this error is not visible:
![image](https://github.com/woocommerce/woocommerce/assets/4463174/733a06ef-4cd1-44d9-b0d8-85309b283d62)
8. Instead, ensure that the Product Rating is visible.





### Changelog entry




<details>

#### Significance




#### Type




#### Message 

CYS - Core: fix Product Rating block renders

#### Comment 

</details>
